storage-fetch/src/main.rs

```diff
@@ -37,23 +34,63 @@ struct Cli {
     #[arg(long, value_parser = parse_duration)]
     timeout: Option<Duration>,
 
-    /// Payload CID
+    /// Deal id
+    #[arg(long)]
+    deal_id: u64,
```

Comment:
nit: this should not be an arg, but just a 'value'.
```suggestion
    deal_id: u64,
```

Reply:
I like it as a named arg. That way you know what the number represents only from reading the command.